.github/workflows/pr-checks.yaml

```diff
@@ -128,4 +128,100 @@ jobs:
             kubectl get pods -A
           
         - name: Run Tests
-          run: make test
+          run: make integration-test
+
+  chainsaw-e2e-tests:
+    name: E2E-Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        adapters: [ "nimbus-kubearmor", "nimbus-netpol", "nimbus-kyverno" ]
+    steps:
+        - name: Checkout source code
+          uses: actions/checkout@v3
+
+        - name: Install helm
+          id: helm
+          uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+          with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+
+        - name: Create k8s Kind Cluster
+          uses: helm/kind-action@v1
+          with:
+            cluster_name: testing
+        
+        - name: Build nimbus image and load in the kind cluster
+          run: |
+            make docker-build
+            kind load docker-image 5gsec/nimbus:latest --name=testing
+
+        - name: Build nimbus-netpol image and load in the kind cluster
+          working-directory: ./pkg/adapter/nimbus-netpol
+          run: |
+            make docker-build
+            kind load docker-image 5gsec/nimbus-netpol:latest --name=testing  
+
+        - name: Build nimbus-kubearmor image and load in the kind cluster
+          working-directory: ./pkg/adapter/nimbus-kubearmor
+          run: |
+            make docker-build
+            kind load docker-image 5gsec/nimbus-kubearmor:latest --name=testing 
+            
+        - name: Build nimbus-kyverno image and load in the kind cluster
+          working-directory: ./pkg/adapter/nimbus-kyverno
+          run: |
+            make docker-build
+            kind load docker-image 5gsec/nimbus-kyverno:latest --name=testing 
+          
+        - name: Install Kubearmor CRDs
+          run: |
+            kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/deployments/CRD/KubeArmorPolicy.yaml
+
+        - name: Install Kyverno CRDs
+          run: |
+            kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/config/crds/kyverno/kyverno.io_clusterpolicies.yaml 
+            kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/config/crds/kyverno/kyverno.io_policies.yaml
```

Comment:
Keep it consistent, use either `kubectl apply` or `kubectl create` but not both.

Reply:
It is Kyverno specific, they specified that we need to use kubectl create, as their annotations are pretty large and exceeding the range of the size buffer.

Reply:
Then use `kubectl create` for KubeArmor as well.

Reply:
done